core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/ccm/upgrade/CcmUpgradeFlowTriggerCondition.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.core.flow2.cluster.ccm.upgrade;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.domain.stack.Stack;
+import com.sequenceiq.cloudbreak.service.stack.StackService;
+import com.sequenceiq.flow.core.FlowTriggerCondition;
+import com.sequenceiq.flow.core.FlowTriggerConditionResult;
+
+@Component
+public class CcmUpgradeFlowTriggerCondition implements FlowTriggerCondition {
```

Comment:
Please add UT.

Reply:
tested in the positiv case, I'll add one for the negativ

Reply:
nope it is here